packages/stores/src/queries-external/github/get-file.ts

```diff
@@ -0,0 +1,68 @@
+import { KVStore } from "@keplr-wallet/common";
+import { HasMapStore } from "@keplr-wallet/stores";
+import { makeObservable } from "mobx";
+
+import { GITHUB_RAW_DEFAULT_BASEURL } from "..";
+import { ObservableQueryExternalBase } from "../base";
+
+export class ObservableQueryFileInner<
+  T
+> extends ObservableQueryExternalBase<T> {
+  constructor(
+    kvStore: KVStore,
+    protected readonly repo: string,
+    protected readonly filePath: string,
+    protected readonly commitHash?: string,
+    protected readonly baseUrl = GITHUB_RAW_DEFAULT_BASEURL
+  ) {
+    super(
+      kvStore,
+      baseUrl,
+      `/${repo}/${commitHash ? commitHash : "main"}/${filePath}`
+    );
+
+    makeObservable(this);
```

Comment:
why would this function need to be observable?

Reply:
Good catch. The `ObservableQueryExternalBase` is already calling the `makeObservable` internally. I'll remove this